### ISSUE TITLE
Boosters at OSIO fails to execute JUnit Tests because of DeploymentException: Process failed to start

### ISSUE THREAD

**Issue by miteshvp** [openshiftio/openshift.io#2827]
### Description
OSIO Boosters fail to execute JUnit tests because of deployment exceptions as seen in the error log below. Currently all the boosters available in OSIO are affected.

#### Steps Performed for Che Terminal
1. Create a quick start project
2. Open Che Workspace
3. Go to Che Terminal
4. cd to the project root dir
5. execute `mvn clean install -Popenshift,openshift-it`

**Error Log**
```
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running io.openshift.boosters.GreetingServiceTest
Resolving 8 out of 636 artifacts
Thu Mar 29 11:38:00 UTC 2018 INFO [org.wildfly.swarm.bootstrap] (main) Dependencies not bundled; resolving from M2REPO.
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 19.123 s <<< FAILURE! - in io.openshift.boosters.GreetingServiceTest
[ERROR] io.openshift.boosters.GreetingServiceTest  Time elapsed: 19.122 s  <<< ERROR!
org.jboss.arquillian.container.spi.client.container.DeploymentException: Process failed to start
Caused by: org.jboss.arquillian.container.spi.client.container.DeploymentException: Process failed to start

[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   GreetingServiceTest.io.openshift.boosters.GreetingServiceTest » Deployment Pro...
[INFO]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 42.279 s
[INFO] Finished at: 2018-03-29T11:38:05+00:00
[INFO] Final Memory: 53M/256M
[INFO] ------------------------------------------------------------------------
[WARNING] The requested profile "openshfit-it" could not be activated because it does not exist.
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.20:test (default-test) on project testbld1: There are test failu
res.
[ERROR]
[ERROR] Please refer to /projects/testbld1/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

**Comment by miteshvp** [comment]
@bartoszmajsak - started seeing this off lately and consistently across all boosters.
@ldimaggi @pmacik - can one of you confirm as part of our E2E tests, whether this is reproducible?

**Comment by miteshvp** [comment]
When I do `mvn help:all-profiles` it does list the profile. See below snippet

```
[INFO] Listing Profiles for Project: io.openshift.booster:testbld1:war:1.0.0-SNAPSHOT
  Profile Id: securecentral (Active: true , Source: settings.xml)
  Profile Id: licenses (Active: false , Source: pom)
  Profile Id: openshift-it (Active: false , Source: pom)
  Profile Id: openshiftio (Active: false , Source: pom)
  Profile Id: release (Active: false , Source: pom)
  Profile Id: openshift (Active: false , Source: pom)
  Profile Id: extra-maven-repository (Active: false , Source: pom)
```

**Comment by pmacik** [comment]
@miteshvp I've noticed that you have a typo in the profile name `openshfit-it` instead of `openshift-it`...

**Comment by miteshvp** [comment]
ok, let me try again. Thanks.

**Comment by joshuawilson** [comment]
@miteshvp The test team doesn't write tests for other teams. The Launcher team owns the boosters. This also needs a `type`.

@gastaldi this looks like something for you.

**Comment by miteshvp** [comment]
This looks to be a typo issue. But I am getting another issue, will create a new one. Closing this one.
